gp-includes/things/note.php

```diff
@@ -0,0 +1,125 @@
+<?php
+/**
+ * Things: GP_Note class
+ *
+ * @package GlotPress
+ * @subpackage Things
+ * @since 3.0.0
+ */
+
+/**
+ * Core class used to implement the notes system.
+ *
+ * @since 3.0.0
+ */
+class GP_Note extends GP_Thing {
+
+	/**
+	 * Name of the database table.
+	 *
+	 * @var string $table_basename
+	 */
+	public $table_basename = 'gp_notes';
+
+	/**
+	 * List of field names for a translation.
+	 *
+	 * @var array $field_names
+	 */
+	public $field_names = array(
+		'id',
+		'original_id',
+		'translation_set_id',
+		'note',
+		'user_id',
+		'date_added',
+		'date_modified',
+	);
+
+	/**
+	 * List of field names which have an integer value.
+	 *
+	 * @var array $int_fields
+	 */
+	public $int_fields = array(
+		'id',
+		'original_id',
+		'translation_set_id',
+		'user_id',
+	);
+
+	/**
+	 * List of field names which cannot be updated.
+	 *
+	 * @var array $non_updatable_attributes
+	 */
+	public $non_updatable_attributes = array( 'id' );
+	
+	public $default_order = 'ORDER BY date_added DESC';
+
+	/**
+	 * Save the note
+	 *
+	 * @since 3.0.0
+	 *
+	 * @param string $note    The new note.
+	 * @param object $translation The translation object.
+	 *
+	 * @return object The output of the query.
+	 */
+	function save( $note, $translation ) {
```

Comment:
The GP_Thing save signature is public with a single $args parameter;
```suggestion
	public function save( $args = null ) {
```
The $note and $translation can live as args and carry through to the create function after you add the other items.